modules/mono/build_scripts/godot_net_sdk_build.py

```diff
@@ -0,0 +1,45 @@
+# Build Godot.NET.Sdk solution
+
+import os
+
+from SCons.Script import Dir
+
+
+def build_godot_net_sdk(source, target, env):
+    # source and target elements are of type SCons.Node.FS.File, hence why we convert them to str
+
+    module_dir = env["module_dir"]
+
+    solution_path = os.path.join(module_dir, "editor/Godot.NET.Sdk/Godot.NET.Sdk.sln")
+    build_config = "Release"
+
+    from .solution_builder import build_solution
+
+    extra_msbuild_args = ["/p:GodotPlatform=" + env["platform"]]
```

Comment:
Why not `-p:` instead of `/p:`? MSBuild seems to support it fine.

Reply:
That's outside of the scope of this PR as we use `/` for MSBuild in all the other places as well. This is something good to consider changing, but we must make sure it doesn't break anything. Was the `-` always supported or was it something introduced after .NET Core?